pkg/storage/pod.go

```diff
@@ -80,8 +82,10 @@ func (s *podStorage) GetMetrics(pods ...apitypes.NamespacedName) ([]api.TimeInfo
 				earliestTimeInfo = ti
 			}
 		}
-		ms[i] = cms
-		tis[i] = earliestTimeInfo
+		if getPodMetrics {
```

Comment:
If this is a bug fix, please add a test.

Reply:
updated